enjoy.py

```diff
@@ -17,7 +18,7 @@
 
 def main():  # noqa: C901
     parser = argparse.ArgumentParser()
-    parser.add_argument("--env", help="environment ID", type=str, default="CartPole-v1")
+    parser.add_argument("--env", help="environment name", type=EnvironmentName, default="CartPole-v1")
```

Comment:
```suggestion
    parser.add_argument("--env", help="environment ID", type=EnvironmentName, default="CartPole-v1")
```

You still pass the id and then convert it to `EnvironmentName`, right?

Reply:
True, I fixed it here and in `{load_from, push_to}_hub.py`